.github/workflows/build-binaries.yml

```diff
@@ -164,6 +164,8 @@ jobs:
             arch: x64
           - target: i686-pc-windows-msvc
             arch: x86
+          - target: aarch64-pc-windows-msvc
+            arch: x64 # not relevant here
```

Comment:
Wait why not?

Reply:
It's only used to install Python, I think?

Reply:
```
      - uses: actions/setup-python@v5
        with:
          python-version: ${{ env.PYTHON_VERSION }}
          architecture: ${{ matrix.platform.arch }}
 ```

Reply:
But why don't we need the ARM Python? Haha

Reply:
So... from what I can see we use Python to

- Transform the README (just needs to match the runner)
- Install the wheel (which we skip for aarch64)

In brief, you can run x86 Python on the x86-64 machine but you can't run the ARM64 Python. It'd just crash?

Reply:
Sneakily.. further down..

```
      - name: "Test wheel"
        if: ${{ !startsWith(matrix.platform.target, 'aarch64') }}
        shell: bash
```

Reply:
Ah ok. So it just needs to match the runner.